README.md

```diff
@@ -1,4 +1,4 @@
-# package-json-to-readme [![Build Status](https://travis-ci.org/zeke/package-json-to-readme.png?branch=master)](https://travis-ci.org/zeke/package-json-to-readme)
+# package-json-to-readme [![Build Status](https://travis-ci.org/VOID404/package-json-to-readme.svg?branch=master)](https://travis-ci.org/VOID404/package-json-to-readme)
```

Comment:
Because if you would not accept this pull request I would just use this repo. Then it would be cheating to have badge from yours project.